# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1245754677 -7200
#      Tue Jun 23 12:57:57 2009 +0200
# Branch release-3-2-x
# Node ID dc56a38b5a64dee2cf19c33232b57702fd984f08
# Parent  6628023f72dc0279a2e4d3163a59dbdaafea8fad
var.m: fix typos (thinkos?) in previous change

diff --git a/scripts/statistics/base/var.m b/scripts/statistics/base/var.m
--- a/scripts/statistics/base/var.m
+++ b/scripts/statistics/base/var.m
@@ -56,17 +56,17 @@ function retval = var (x, opt, dim)
     opt = 0;
   endif
 
   sz = size (x);
   n = sz(dim);
   if (isempty (x))
     ## FIXME -- is there a way to obtain these results without all the
     ## special cases?
-    if (ndim (x) == 2 && sz(0) == 0 && sz(1) == 0)
+    if (ndims (x) == 2 && sz(1) == 0 && sz(2) == 0)
       retval = NaN;
     else
       sz(dim) = 1;
       if (n == 0)
 	if (prod (sz) == 0)
 	  retval = zeros (sz);
 	else
 	  retval = NaN (sz);
